### ISSUE TITLE
wrong output of exported onnx model when batch_size > 1

### ISSUE THREAD

**Issue by johnrock19981** [open-mmlab/mmdetection#7160]
Thanks for your error report and we appreciate it a lot.

**Checklist**

1. I have searched related issues but cannot get the expected help.
2. I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
3. The bug has not been fixed in the latest version.

**Describe the bug**

I exported mask-rcnn with swin-t backbone trained on a custom dataset to onnx, the outputs is wrong when input batch size is greater than 1 ([2, 3, 500, 500] for example), both `tools/deployment/pytorch2onnx.py` and mmdeploy's deploy script are tested. When exported with mmdetection's pytorch2onnx, the output gives wrong bbox and segm; when exported with mmdeploy,  there is no bbox or segm found in output.

command used to export model with mmdetection
```
python tools/deployment/pytorch2onnx.py [CONFIG_FILE] [CHECKPOINT_FILE] --input-img [INPUT_IMG] --dynamic-export --verify
```

command used to export model with mmdeploy
```
python $MMDEPLOY_ROOT/tools/deploy.py $MMDEPLOY_ROOT/configs/mmdet/detection/detection_onnxruntime_dynamic.py [CONFIG_FILE] [CHECKPOINT_FILE]
```

**Reproduction**

1. What command or script did you run?

onnxruntime version: 1.10.0
mmcv-full version: 1.4.4
mmdetection version: master

Config file used: [configs/swin/mask_rcnn_swin-t-p4-w7_fpn_ms-crop-3x_coco.py](https://github.com/open-mmlab/mmdetection/blob/master/configs/swin/mask_rcnn_swin-t-p4-w7_fpn_ms-crop-3x_coco.py)

I tried to modify the onnx model in test script to accept dynamic batch size by changing this [line](https://github.com/open-mmlab/mmdetection/blob/98949809b7179fab9391663ee5a4ab5978425f90/mmdet/core/export/model_wrappers.py#L131) to `input_data = torch.concat(imgs)`, it gives the same wrong result in my deployment environment.

2. Did you make any modifications on the code or config? Did you understand what you have modified?
3. What dataset did you use?

**Environment**

1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment information and paste it here.
2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**
If applicable, paste the error trackback here.

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by hhaAndroid** [comment]
@johnrock19981 Sorry. Swin model is not supported yet.

**Comment by ZwwWayne** [comment]
As [MMDeploy](https://github.com/open-mmlab/mmdeploy) is released. MMDet will no longer maintain the feature of model export. Please use MMDeploy and create a feature request if you wish to export swin transformer.
